kinto/core/resource/__init__.py

```diff
@@ -1087,7 +1087,9 @@ def _build_pagination_token(self, sorting, last_record, offset):
         }
 
         for field, _ in sorting:
-            token['last_record'][field] = last_record[field]
+            last_value = find_nested_value(last_record, field)
+            if last_value is not None:
+                token['last_record'][field] = last_value
```

Comment:
What happens if `last_value` is None ? There is no pagination ? Would that solve #816 ?